src/definitions/modules/search.less

```diff
@@ -520,6 +520,16 @@
   }
 }
 
+/*--------------
+    Squared
+---------------*/
+
+& when (@variationSearchSquared) {
+  .ui.squared.search .prompt {
+    border-radius: .28571429rem;
```

Comment:
- Please exchange the hardcoded value by a variable `@promptBorderRadiusSquared`
- In the file `search.variables` create an entry `@promptBorderRadiusSquared: @absoluteBorderRadius;`

Reply:
> Please create a proper PR description (yes, your code is obvious, but we usually don't guess functionality just by viewing the code)

Ehm... how to add it now? General speaking, how to edit a PR here, including file changes?

> At least a jsfiddle to prove your change works should be provided and always helpful.
> Friendlier for reviewers 

Do you mean [this](https://jsfiddle.net/lubber/3dowLgsx/2)?

Reply:
I saw a lot of occurences of `.28571429rem` in the source code of the project, so I thought that it could be right.

Reply:
where did you see those raw values? Raw values are only inside of generated CSS files, but should never appear inside of .less files

Reply:
You are right, sorry!